pytab/core.py

```diff
@@ -361,6 +361,21 @@ def cli(
     click.echo("| Obtaining System Information...", nl=False)
     system_info = hwi.get_system_info()
     click.echo(" success!")
+    click.echo("| Detected System Config:")
+    click.echo(f"|   OS: {system_info['os']['pretty_name']}")
+    for cpu in system_info['cpu']:
+        click.echo(f"|   CPU: {cpu['product']}")
+        click.echo(f"|     Threads: {cpu['cores']}")
+        click.echo(f"|     Arch: {cpu['architecture']}")
+    
+    click.echo("|   RAM:")
+    for ram in system_info['memory']:
+        click.echo(f"|     {ram['id']}: {ram['vendor']} {ram['size']} {ram['units']}")
+
+    click.echo("|   GPU(s):")
+    for gpu in system_info['gpu']:
+        click.echo(f"|     {gpu['id']}: {gpu['product']}")
+    click.pause("Press any key to continue")
```

Comment:
Offering the option to continue/ abort might not be a bad idea though.
(Sorry, I know I original disliked it...)

Because the user might just see "Press key to continue" and just press a key.
Having something like "Confirm Hardware and continue (y/N)" with e.g. `click.confirm` might be a good Idea. 

I dont REALY care about how we do this, since we can just change it at any point!

Reply:
I think the abort option can be added in a future PR